src/main/target/MAMBAF411/target.h

```diff
@@ -32,7 +32,8 @@
 #define BEEPER_PIN              PB2
 #define BEEPER_INVERTED
 
-#define ENABLE_DSHOT_DMAR       true
+#undef  ENABLE_DSHOT_DMAR
```

Comment:
Can you please remove this - this will probably break all kinds of configurations that work now.

Reply:
@mikeller I’ve mapped out all dma streams including led for all of these targets and there were no conflicts. Which types of breakage are you worried about?

Reply:
I can add the dma stream uses for each target to this PR if it helps.

Reply:
Have you tested all of the DMA streams for conflicts with resources like SPI on the actual hardware?

Reply:
Here's what I have done:

Matekf411rx:
```
    DEF_TIM(TIM2, CH3, PB10, TIM_USE_MOTOR, 0, 0), // S1_OUT - DMA1_ST1
    DEF_TIM(TIM4, CH1, PB6,  TIM_USE_MOTOR, 0, 0), // S2_OUT - DMA1_ST0
    DEF_TIM(TIM4, CH2, PB7,  TIM_USE_MOTOR, 0, 0), // S3_OUT - DMA1_ST3
    DEF_TIM(TIM4, CH3, PB8,  TIM_USE_MOTOR, 0, 0), // S4_OUT - DMA1_ST7
    DEF_TIM(TIM5, CH1, PA0,  TIM_USE_LED,   0, 0),     // 2812LED - DMA1_ST2
   ADC uses DMA2_ST0
```

MatekF411:

```
    DEF_TIM(TIM3, CH1, PB4,  TIM_USE_MOTOR,  0, 0), // S1_OUT - DMA1_ST4
    DEF_TIM(TIM3, CH2, PB5,  TIM_USE_MOTOR,  0, 0), // S2_OUT - DMA1_ST5
    DEF_TIM(TIM4, CH1, PB6,  TIM_USE_MOTOR,  0, 0), // S3_OUT - DMA1_ST0
    DEF_TIM(TIM4, CH2, PB7,  TIM_USE_MOTOR,  0, 0), // S4_OUT - DMA1_ST3
    DEF_TIM(TIM2, CH2, PB3,  TIM_USE_MOTOR, 0, 0), // S5 - DMA1_ST6
    DEF_TIM(TIM2, CH3, PB10, TIM_USE_MOTOR, 0, 0), // S6 - DMA1_ST1
    DEF_TIM(TIM1, CH1, PA8,  TIM_USE_LED,   0, 0), // LED - DMA2_ST6
   ADC uses DMA2_ST0
```

Mambaf411:

```
    DEF_TIM(TIM2,   CH2,  PB3,  TIM_USE_MOTOR, 0, 0 ), // DMA1_ST6
    DEF_TIM(TIM3,   CH1,  PB4,  TIM_USE_MOTOR, 0, 0 ), //  DMA1_ST4
    DEF_TIM(TIM4,   CH1,  PB6,  TIM_USE_MOTOR, 0, 0 ), // DMA1_ST0
    DEF_TIM(TIM4,   CH2,  PB7,  TIM_USE_MOTOR, 0, 0 ), // DMA1_ST3
    DEF_TIM(TIM2,   CH1,  PA15, TIM_USE_ANY,   0, 0 ), // DMA1_ST5
    DEF_TIM(TIM10,  CH1,  PB8,  TIM_USE_ANY,   0, 0 ), // NONE

    DEF_TIM(TIM1,   CH1,  PA8,  TIM_USE_LED,   0, 0 ), // DMA2_ST6
   ADC uses DMA2_ST0
```

None of these FCs define OSD DMA or have an sdcard slot. I have personally flown the crazybeef4 which is a matekf411rx variant.

I don't think we have any other potential users of dma on these boards. I have strong confidence that they will in fact will work great with timer based dshot.

Each of these boards will either break led or soft serial with dshot_bitbang so defaulting them to dshot_bitbang=off and dshot_burst=off improves the situation over what's in rc2.

I don't have each of these boards so I can't test them all. That's what I was trying to accomplish in the next RC.